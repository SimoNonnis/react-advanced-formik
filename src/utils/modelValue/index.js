function modelValue(value) {
  return value === null ? value : value.toString();
}

export default modelValue;
