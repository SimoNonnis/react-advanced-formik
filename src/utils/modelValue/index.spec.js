/* eslint-env jest */
import modelValue from './';

describe('Test modelValue()', () => {
  it('should return null if value is null', () => {
    expect(modelValue(null)).toMatchSnapshot();
  });
  it('should transform the value to a string', () => {
    expect(modelValue(true)).toMatchSnapshot();
  });
});
