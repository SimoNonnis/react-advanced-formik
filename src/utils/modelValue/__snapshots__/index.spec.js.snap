// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test modelValue() should return null if value is null 1`] = `null`;

exports[`Test modelValue() should transform the value to a string 1`] = `"true"`;
